frontend/src/components/cluster/Overview.tsx

```diff
@@ -116,15 +116,44 @@ function EventsSection() {
         columns={events ? [
           {
             label: 'Type',
-            getter: event => event.involvedObject.kind
+            getter: event => event.involvedObject.kind,
+            sort: (e1: Event, e2: Event) => {
+              if (e1.involvedObject.kind < e2.involvedObject.kind) {
+                return 1;
+              } else if (e1.involvedObject.kind > e2.involvedObject.kind) {
+                return -1;
+              } else {
+                return 0;
+              }
+            }
           },
           {
             label: 'Name',
             getter: event => event.involvedObject.name,
+            sort: (e1: Event, e2: Event) => {
+              const e1Name = e1.involvedObject.name.toUpperCase();
+              const e2Name = e2.involvedObject.name.toUpperCase();
+              if (e1Name < e2Name) {
+                return 1;
+              } else if (e1Name > e2Name) {
+                return -1;
+              } else {
+                return 0;
+              }
+            }
           },
           {
             label: 'Age',
             getter: event => timeAgo(event.metadata.creationTimestamp),
+            sort: (e1: Event, e2: Event) => {
+              if (e1.metadata.creationTimestamp < e2.metadata.creationTimestamp) {
+                return 1;
+              } else if (e1.metadata.creationTimestamp > e2.metadata.creationTimestamp) {
+                return -1;
+              } else {
+                return 0;
+              }
+            }
```

Comment:
This one we need to override (because the getter will return `timeAgo` which is not sortable). Note that the function can be highly simplified as `(e1: Event, e2: Event) => e2.metadata.creationTimestamp - e1.metadata.creationTimestamp`.